src/ensembl/src/content/app/browser/browserReducer.ts

```diff
@@ -51,10 +52,23 @@ export function browserEntity(
   action: ActionType<typeof browserActions>
 ): BrowserEntityState {
   switch (action.type) {
+    case getType(browserActions.setDataFromUrl): {
+      const { activeGenomeId, activeEnsObjectId } = action.payload;
+      const newState = {
+        ...state,
+        activeGenomeId
+      };
+      if (!activeEnsObjectId) {
+        delete newState.activeEnsObjectIds[activeGenomeId];
```

Comment:
I used a debugger here but it looks like whenever this gets executed, there are no matching entries to delete.
Do you know when this will actually delete an entry?

Reply:
Good question; and I can't remember my thinking at the moment. I'll remove this check then.